### ISSUE TITLE
binding/abl-imatinib example broken

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank-examples#61]
Currently, the `binding/abl-imatinib/` example does not work with the latest `yank 0.17.0`:
```
[chodera@lilac:abl-imatinib]$ ./run-explicit.sh 
Running simulation...
found unknown parameter switch_experiment_every
Traceback (most recent call last):
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/experiment.py", line 1014, in _validate_options
    special_conversions=special_conversions)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/utils.py", line 1151, in validate_parameters
    raise TypeError("found unknown parameter {}".format(', '.join(diff)))
TypeError: found unknown parameter switch_experiment_every

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/chodera/miniconda/bin/yank", line 11, in <module>
    load_entry_point('yank==0.17.0', 'console_scripts', 'yank')()
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/cli.py", line 71, in main
    dispatched = getattr(commands, command).dispatch(command_args)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/commands/script.py", line 110, in dispatch
    yaml_builder = ExperimentBuilder(script=yaml_path, job_id=job_id, n_jobs=n_jobs)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/experiment.py", line 558, in __init__
    self.parse(script)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/experiment.py", line 648, in parse
    validate_general_options=True))
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/experiment.py", line 1016, in _validate_options
    raise YamlParseError(str(e))
yank.experiment.YamlParseError: found unknown parameter switch_experiment_every
Analyzing data...
2017-09-15 23:25:30,311: ERROR - yank.analyze - Cannot find analysis.yaml script in experiments
Traceback (most recent call last):
  File "/home/chodera/miniconda/bin/yank", line 11, in <module>
    load_entry_point('yank==0.17.0', 'console_scripts', 'yank')()
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/cli.py", line 71, in main
    dispatched = getattr(commands, command).dispatch(command_args)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/commands/analyze.py", line 81, in dispatch
    analyze.analyze_directory(args['--store'])
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/analyze.py", line 1378, in analyze_directory
    raise RuntimeError(err_msg)
RuntimeError: Cannot find analysis.yaml script in experiments
```

**Comment by jchodera** [comment]
Eliminating the `switch_experiment_every` line in the YAML file gives
```
[chodera@lilac:abl-imatinib]$ rm -rf explicit; ./run-explicit.sh 
Running simulation...
2017-09-16 00:03:46,047: Setting CUDA platform to use precision model 'mixed'.
2017-09-16 00:03:46,063: Single node: executing <function ExperimentBuilder._check_resume at 0x2b5bb593ad08>
2017-09-16 00:03:46,065: Single node: executing <function ExperimentBuilder._setup_experiments at 0x2b5bb593af28>
2017-09-16 00:03:46,066: Setting up the systems for Abl, STI_0 and pme
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
/data/chodera/schrodinger/schrodinger2017-2-advanced/utilities/structconvert: line 54: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
/data/chodera/schrodinger/schrodinger2017-2-advanced/epik: line 54: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
/data/chodera/schrodinger/schrodinger2017-2-advanced/utilities/maesubset: line 54: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
/data/chodera/schrodinger/schrodinger2017-2-advanced/utilities/structconvert: line 54: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
/data/chodera/schrodinger/schrodinger2017-2-advanced/utilities/structconvert: line 54: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
/data/chodera/schrodinger/schrodinger2017-2-advanced/utilities/proplister: line 54: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
2017-09-16 00:04:27,611: 4-[(4-methylpiperazin-4-ium-1-yl)methyl]-~{N}-[4-methyl-3-[[4-(3-pyridyl)pyrimidin-2-yl]amino]phenyl]benzamide
Traceback (most recent call last):
  File "/home/chodera/miniconda/bin/yank", line 11, in <module>
    load_entry_point('yank==0.17.0', 'console_scripts', 'yank')()
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/cli.py", line 71, in main
    dispatched = getattr(commands, command).dispatch(command_args)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/commands/script.py", line 113, in dispatch
    yaml_builder.run_experiments()
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/experiment.py", line 712, in run_experiments
    group_nodes=processes_per_experiment)
  File "/home/chodera/miniconda/lib/python3.5/contextlib.py", line 77, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/openmoltools/utils.py", line 286, in temporary_cd
    yield
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/experiment.py", line 694, in run_experiments
    self._setup_experiments()
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/mpi.py", line 250, in _wrapper
    return run_single_node(rank, task, *args, **kwargs)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/mpi.py", line 199, in run_single_node
    result = task(*args, **kwargs)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/experiment.py", line 1753, in _setup_experiments
    self._db.get_system(system_id)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/pipeline.py", line 994, in get_system
    0, system_parameters, solvent_id, ligand_id)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/pipeline.py", line 1455, in _setup_system
    warnings = tleap.run()
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/utils.py", line 1738, in run
    return re.findall('WARNING: (.+)', leap_output)
  File "/home/chodera/miniconda/lib/python3.5/contextlib.py", line 77, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/mdtraj/utils/contextmanagers.py", line 44, in enter_temp_directory
    yield
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/utils.py", line 1735, in run
    raise RuntimeError(error_msg + ' Check log file {}'.format(log_path))
RuntimeError: Could not create one of the system files. Check log file /lila/home/chodera/github/yank-examples/examples/binding/abl-imatinib/explicit/setup/systems/Abl-STI_STI0/solvent.leap.log
Analyzing data...
2017-09-16 00:04:31,419: ERROR - yank.analyze - Cannot find analysis.yaml script in experiments
Traceback (most recent call last):
  File "/home/chodera/miniconda/bin/yank", line 11, in <module>
    load_entry_point('yank==0.17.0', 'console_scripts', 'yank')()
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/cli.py", line 71, in main
    dispatched = getattr(commands, command).dispatch(command_args)
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/commands/analyze.py", line 81, in dispatch
    analyze.analyze_directory(args['--store'])
  File "/home/chodera/miniconda/lib/python3.5/site-packages/yank/analyze.py", line 1378, in analyze_directory
    raise RuntimeError(err_msg)
RuntimeError: Cannot find analysis.yaml script in experiments
```
The LEaP output file is full of 
```
Could not find bond parameter for: EP - OW
```
which suggests we are trying to use TIP4P-Ew in an improper way.

**Comment by jchodera** [comment]
Actually, the parameters block specifies TIP3P:
```
systems:
  Abl-STI:
    receptor: Abl
    ligand: STI
    solvent: pme
    leap:
      parameters: [leaprc.protein.ff14SB, leaprc.gaff, leaprc.water.tip3p]
```
Despite this, the LEaP output file suggests that YANK is trying to solvate the system with a TIP4P-Ew water box:
```
> # Solvate systems
> solvateBox STI_0 TIP4PEWBOX 9.0 iso
  Solute vdw bounding box:              24.913 11.667 9.312
  Total bounding box for atom centers:  42.913 42.913 42.913
      (box expansion for 'iso' is 127.3%)
  Solvent unit box:                     18.860 18.867 18.860
The number of boxes:  x= 3  y= 3  z= 3
```

**Comment by jchodera** [comment]
Looks like we need to add a [`solvent_model:`](http://getyank.org/0.17.0/yamlpages/solvents.html#solvent-model).

**Comment by jchodera** [comment]
Looks like we needed to change `switch_experiment_every` to [`switch_experiment_interval`](http://getyank.org/0.17.0/yamlpages/options.html#switch-experiment-interval).

**Comment by jchodera** [comment]
I wonder if we can re-enable travis nightlies at this point to catch these problems when we make upstream changes.

**Comment by jchodera** [comment]
This should be fixed by #64?
